FindEquilibrium.java

```diff
@@ -0,0 +1,34 @@
+
+public class FindEquilibrium {
+
+	public static int findEquilibriumIndex(int[] inputArr){
+		
+		if(inputArr == null || inputArr.length <= 2){
+			return 0;
+		}
+		int left = 0, right = 0 , total = 0;
+		
+		for(int i = 0; i < inputArr.length ; i++){
+			total += inputArr[i];
+		}
+		
+		for(int i = 0 ; i< inputArr.length ; i++){
+			left += inputArr[i];
+			
+			right = total - left - inputArr[i+1];
```

Comment:
does the (i+1) will throw outofboundsexception?
{1,2,3,4,5} - i ran for this input and it dint work..pls check. Also see if you can solve the problem in 1 iteration instead of 2, still you solved in O(n) which is good... but giv a try in one iteration..